### ISSUE TITLE
About groundtruth udf in loss function

### ISSUE THREAD

**Issue by crabcoding** [LIA-DiTella/nsdf_editing#3]
In the loss function, it seems that gt udf is used as a supervision to train the network. Is it fair?

![loss](https://github.com/user-attachments/assets/e8518ea2-5595-4c1b-8dba-037693ceeb46)
![loss2](https://github.com/user-attachments/assets/293ad1f2-7359-4556-ba19-353c91fed609)

**Comment by crabcoding** [comment]
In your work, normals of point clouds are required as inputs, while in CAP-UDF, normals are not required. I wonder whether the comparisons are fair or not.

**Comment by miguef98** [comment]
Hi, thanks for the question!

You're right—our method uses ground truth distances for supervision. However, our approach differs from CAP-UDF in that we focus on open surface representation, not point-cloud reconstruction. We chose to compare our method to CAP-UDF because, to our knowledge, it was the leading method not only for point-cloud reconstruction but also for open surface representation.

While we used ground truth distances in our paper, it's important to note that the sampling schemes employed in point-cloud reconstruction methods aren't orthogonal to ours. A distance approximation method could be developed for the training stage.

Hope this helps clarify things!
